src/js/msp/MSPHelper.js

```diff
@@ -2631,6 +2639,16 @@ MspHelper.prototype.sendLedStripModeColors = function(onCompleteCallback) {
     }
 };
 
+MspHelper.prototype.sendLedStripConfigValues = function(onCompleteCallback) {
+    {
+        const buffer = [];
+        buffer.push8(FC.LED_CONFIG_VALUES.brightness);
+        buffer.push8(FC.LED_CONFIG_VALUES.rainbow_delta);
+        buffer.push8(FC.LED_CONFIG_VALUES.rainbow_freq);
+        MSP.send_message(MSPCodes.MSP_SET_LED_STRIP_CONFIG_VALUES, buffer, false, onCompleteCallback);
+    }
+};
```

Comment:
Redundant curly braces. Functions have their own lexical scope.
```suggestion
MspHelper.prototype.sendLedStripConfigValues = function(onCompleteCallback) {
    const buffer = [];
    buffer.push8(FC.LED_CONFIG_VALUES.brightness);
    buffer.push8(FC.LED_CONFIG_VALUES.rainbow_delta);
    buffer.push8(FC.LED_CONFIG_VALUES.rainbow_freq);
    MSP.send_message(MSPCodes.MSP_SET_LED_STRIP_CONFIG_VALUES, buffer, false, onCompleteCallback);
};
```